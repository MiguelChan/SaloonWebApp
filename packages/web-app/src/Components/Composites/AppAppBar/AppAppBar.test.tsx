import React from 'react';
import {
  render,
} from '@testing-library/react';
import { AppAppBar } from '.';

describe('AppAppBar', () => {
  it('Should match snapshot', () => {
    const {
      container,
    } = render(<AppAppBar />);
    expect(container).toMatchSnapshot();
  });
});