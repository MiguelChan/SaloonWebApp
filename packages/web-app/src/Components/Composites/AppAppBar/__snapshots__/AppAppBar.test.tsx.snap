// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppAppBar Should match snapshot 1`] = `
<div>
  <div>
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-1hc7nu0-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-aqslzj-MuiToolbar-root"
      >
        <div
          class="MuiBox-root css-1rr4qq7"
        />
        <a
          class="MuiTypography-root MuiTypography-h6 MuiLink-root MuiLink-underlineNone css-2r3fq0-MuiTypography-root-MuiLink-root"
          href="#"
        >
          Lashair by Samy
        </a>
        <div
          class="MuiBox-root css-urdvyo"
        >
          <a
            class="MuiTypography-root MuiTypography-h6 MuiLink-root MuiLink-underlineNone css-1we74st-MuiTypography-root-MuiLink-root"
            href="#"
          >
            Iniciar Sesion
          </a>
          <a
            class="MuiTypography-root MuiTypography-h6 MuiLink-root MuiLink-underlineNone css-1apndo0-MuiTypography-root-MuiLink-root"
            href="#"
          >
            Registrarse
          </a>
        </div>
      </div>
    </header>
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-gppu2o-MuiToolbar-root"
    />
  </div>
</div>
`;
